api/external/infra_proxy/request/databags.proto

```diff
@@ -4,18 +4,18 @@ package chef.automate.api.infra_proxy.request;
 option go_package = "github.com/chef/automate/api/external/infra_proxy/request";
 
 message DataBags {
-    // Id of the Org.
+    // ID of the chef organization.
```

Comment:
I think I would suggest capitalizing Chef, being a proper name, here and all other similar comments.
```suggestion
    // ID of the Chef organization.
```